File: easy100/001/main.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    int ans = 0, taps = 1;
    while (taps < b) {
        taps += a - 1;
        ans++;
    }
    cout << ans << endl;
    return 0;
}

File: easy100/001/main.go
package main

import "fmt"

func main() {
	var a, b int
	fmt.Scan(&a, &b)
	var ans, taps int = 0, 1
	for taps < b {
		taps += a - 1
		ans++
	}
	fmt.Println(ans)
}
